pkg/server/config.go

```diff
@@ -71,6 +71,9 @@ type Config struct {
 	// CAKeyType is the key type used for the X509 and JWT signing keys
 	CAKeyType keymanager.KeyType
```

Comment:
I wonder if this should be renamed to X509CAKeyType? If so, maybe there should be some backwards compatibility with the `ca_key_type` config field?